docs/user-interface/images/app-icons.md

```diff
@@ -1,83 +1,215 @@
+
+```xml
+<ItemGroup>
+    <MauiIcon Include="Resources\AppIcon\appicon.svg" ForegroundFile="Resources\AppIcon\appiconfg.svg" ForegroundScale="0.65" Color="#512BD4" />
+</ItemGroup>
+```
+
+The `ForegroundScale` attribute can be optionally specified to change the scaling of the foreground layer being rendered over the background layer.
+
+# [iOS & macOS](#tab/ios)
+
+The app icon defined by your .NET MAUI app is used to generate an asset catalog icon set for both iOS and macOS platforms. The name of the icon set is defined in the plist file, which on iOS is located at _Platforms\\iOS\\Info.plist_. For macOS, the plist file is located at _Platforms\\MacCatalyst\\Info.plist_.
+
+> [!TIP]
+> To edit the plist file in Visual Studio, the plist editor presented when you double click on the plist file may not work. You may need to right-click the file in the **Solution Explorer** pane, and select **Open With...**. Next, choose **XML (Text) Editor** and select **OK**.
+
+The plist file contains a `<key>XSAppIconAssets</key>` entry, with a corresponding `<string>` node defined right after it. The value of this `<string>` node follows this format: `Assets.xcassets/{name}.appiconset` The value for `{name}` is derived from the .NET MAUI project file's `<MauiIcon>` item, specifically the file name defined by the `Include` attribute, without its path or extension.
+
+Consider the following example, which defines the resource `Resources\AppIcon\healthapp.png` as the icon:
+
+```xml
+<ItemGroup>
+    <MauiIcon Include="Resources\AppIcon\healthapp.png" ForegroundFile="Resources\AppIcon\appiconfg.svg" TintColor="Yellow" />
+</ItemGroup>
+```
+
+The transformed name, the resource without the path or extension, is `healthapp`. The value in the manifest would be `Assets.xcassets/healthapp.appiconset`, as demonstrated in the following XML:
+
+```xml
+<key>XSAppIconAssets</key>
+<string>Assets.xcassets/healthapp.appiconset</string>
+```
+
+> [!TIP]
+> Instead of creating new image files for the icon, simply replace the two image files provided by the .NET MAUI template: _Resources\\AppIcon\\appicon.svg_ for the background and _Resources\\AppIcon\\appiconfg.svg_ for the foreground.
 
-On Android, drawable XML resources and bitmaps are generated from the app icon. An adaptive launcher icon is generated when the `ForegroundFile` attribute is defined.
+# [Windows](#tab/windows)
 
-# [iOS](#tab/ios)
+There aren't any other settings to configure for Windows.
 
-On iOS, an asset catalog icon set is generated from the app icon.
+The app icon defined by your .NET MAUI app is used to generate an asset catalog icon set.
```

Comment:
Is it definitely called an asset catalog icon set on Windows? That's iOS terminology.